propagator/xray/lib/opentelemetry/propagator/xray/text_map_extractor.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+# Copyright 2020 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# OpenTelemetry is an open source observability framework, providing a
+# general-purpose API, SDK, and related tools required for the instrumentation
+# of cloud-native software, frameworks, and libraries.
+#
+# The OpenTelemetry module provides global accessors for telemetry objects.
+# See the documentation for the `opentelemetry-api` gem for details.
+module OpenTelemetry
+  # Namespace for OpenTelemetry propagator extension libraries
+  module Propagator
+    # Namespace for OpenTelemetry XRay propagation
+    module XRay
+      # Extracts context from carriers in the xray single header format
+      class TextMapExtractor
+        include Context::Propagation::DefaultGetter
```

Comment:
This isn't a thing anymore. See https://github.com/open-telemetry/opentelemetry-ruby/blob/main/api/lib/opentelemetry/baggage/propagation/text_map_extractor.rb as an example.